project1/application.py

```diff
@@ -25,6 +25,12 @@
 def index():
     return render_template("index.html")
 
-@app.route("/register")
+@app.route("/register",methods =["GET","POST"])
 def register():
+    var=request.form.get("email")
+    print(var)
+    var1=request.form.get("psw")
+    print(var1)
```

Comment:
Add these lines in the if condition. As these will still be implemented in 'GET' request. As it wont have any data, it might throw some errors.

Reply:
ok will do that